tests/src/test/java/alluxio/master/JournalIntegrationTest.java

```diff
@@ -455,6 +460,32 @@ private void manyFileTestUtil() throws Exception {
   }
 
   /**
+   * Tests the situation where a checkpoint mount entry is replayed by a standby master.
+   *
+   * @throws Exception on error
+   */
+  @Test
+  public void mountEntryCheckpointTest() throws Exception {
+    AlluxioURI mountUri = new AlluxioURI("/local_mnt/");
+    AlluxioURI ufsUri = new AlluxioURI(mTestFolder.newFolder("test_ufs").getAbsolutePath());
+
+    // Create a mount point, which will journal a mount entry.
+    mFileSystem.mount(mountUri, ufsUri);
+    mLocalAlluxioCluster.stopFS();
+
+    // Start a leader master, which will create a new checkpoint, with a mount entry.
+    FileSystemMaster fsMaster = MasterTestUtils.createLeaderFileSystemMasterFromJournal();
+    fsMaster.stop();
+
+    // Start a standby master, which will replay the mount entry from the checkpoint.
+    fsMaster = MasterTestUtils.createStandbyFileSystemMasterFromJournal();
+    // The sleep is necessary for the journal thread to replay the checkpoint, before trying to read
+    // the master.
+    CommonUtils.sleepMs(2000);
```

Comment:
Is there a way to get a callback when the checkpoint replay is done? I think this could lead to flakey tests?